addons/pos_sale/static/src/overrides/models/pos_store.js

```diff
@@ -151,7 +151,7 @@ patch(PosStore.prototype, {
             if (product_unit && !product_unit.is_pos_groupable) {
                 let remaining_quantity = newLine.qty;
                 while (!this.env.utils.floatIsZero(remaining_quantity)) {
-                    const splitted_line = this.models["pos.order.line"].create(newLineValues);
+                    const splitted_line = this.models["pos.order.line"].create({...newLineValues, });
```

Comment:
```suggestion
                    const splitted_line = this.models["pos.order.line"].create({...newLineValues });
```
Could you remove the trailing comma ?

Reply:
i added because runbot/ci failed with suggestion (from eslint test) to add the comma.